StudyBuddy/Controllers/ProfileController.cs

```diff
@@ -66,7 +66,7 @@ public IActionResult UserProfile(string id)
     public IActionResult CreateProfile() => View();
 
     [HttpPost]
-    public async Task<IActionResult> SaveProfile(string name, string birthdate, string subject, IFormFile? avatar, string? markdownContent, List<string> hobbies)
+    public async Task<IActionResult> SaveProfile(string name, string birthdate, string subject, IFormFile? avatar, string? markdownContent, List<string> hobbies, String? longitude, String? latitude)
```

Comment:
In C#, the string keyword is an alias for [String](https://learn.microsoft.com/en-us/dotnet/api/system.string); therefore, String and string are equivalent. It's recommended to use the provided alias string as it works even without using System;.